appveyor.yml

```diff
@@ -185,6 +185,9 @@
 
             if ($env:BOND_BUILD -eq "C++") {
 
+                $env:PreferredToolArchitecture = "x64"
```

Comment:
I have a hunch that it doesn't persist to the build_script stage and I want to see if I get any difference when the CI completes. I had problems with line 129 that's why I had to move it down to right before running cmake since setting it before wouldn't work. If it does work I'll move it before the if logic but still during build_script and not during before_build.